joomla/joomla-cms
# [4.0] Article batch operation fixes

Pull Request for Issue #21285.

### Summary of Changes
Classes which were not defined but were in use became defined by a simple syntax changing. The namespaces had to be set global by putting a backslash before the classnames. Affected classes: JLoader, FieldsHelper 

JEventDispatcher is deprecated, so the event has to be triggered by getting the main application from Factory.

### Testing Instructions
Select an article clicking on the check box at the beginning of the line. Click on the "Batch"-button and select a category in the field below "To Move or Copy your selection please select a Category.". Check out any radio button and click "Process".


### Documentation Changes Required
No changes required.
